frontend/src/components/Authentification/AuthCheck.vue.js

```
@@ -0,0 +1,52 @@
+const ready = ref(false);
+onMounted(async () => {
+    try {
+        // If middleware redirected you to OIDC already, this won’t run until after login
+        await store.loadBootstrap();
+    }
+    finally {
+        ready.value = true;
+    }
+});
+const isAuth = isAuthenticated;
+debugger; /* PartiallyEnd: #3632/scriptSetup.vue */
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)